### ISSUE TITLE
define the workshop directory

### ISSUE THREAD

**Issue by goi42** [datacarpentry/python-ecology-lesson#529]
In [Starting With Data](https://datacarpentry.org/python-ecology-lesson/02-starting-with-data/index.html), learners are instructed to navigate to "the workshop directory". This phrase has not been defined in the [Setup](https://datacarpentry.org/python-ecology-lesson/setup.html) or any of the earlier lessons, as far as I can tell.

My best guess is that this refers to the [zip file](https://datacarpentry.org/python-ecology-lesson/data/portal-teachingdb-master.zip) that can be downloaded from the Setup page, but this doesn't follow the guidance in [Before we start](https://datacarpentry.org/python-ecology-lesson/00-before-we-start/index.html), which suggests data be stored in the `data/` subdirectory.

I *think* the lesson wants the learners to create an empty directory for the workshop, create a subdirectory inside it called `data/`, then move the [downloaded files](https://datacarpentry.org/python-ecology-lesson/data/portal-teachingdb-master.zip) into it. If so, learners should be stepped through this process explicitly to avoid confusion.

**Comment by goi42** [comment]
https://github.com/datacarpentry/python-ecology-lesson/blob/73b8d44446812d8ec45a054077dcfb34724de946/_episodes/04-data-types-and-format.md?plain=1#L406

The [Writing Out Data to CSV](https://datacarpentry.org/python-ecology-lesson/04-data-types-and-format/index.html#writing-out-data-to-csv) exercise fails with an `OSError` because the `data_output/` directory does not exist.

Either this subdirectory should be created as part of defining the workshop directory in an earlier lesson (as described above), or instructions should be added for how to create a subdirectory.

**Comment by goi42** [comment]
Lesson 5 repeats the [Writing Out Data to CSV](https://github.com/datacarpentry/python-ecology-lesson/blob/73b8d44446812d8ec45a054077dcfb34724de946/_episodes/05-merging-data.md#writing-out-data-to-csv) exercise using concatenated data frames, but this time stores the output in the `data/` subdirectory instead of `data_output`.

It would be better to be consistent across these exercises, especially since the [Before we start](https://datacarpentry.org/python-ecology-lesson/00-before-we-start/index.html) lesson makes a point of maintaining good directory organization.

**Comment by btovar** [comment]
@goi42 I agree with you about the consistency of `data` and `data_output`.  Would you like to make a pull request to fix this one?

**Comment by jas58** [comment]
Yes, please!  This was a nightmare today.
It's more than a typo.  I'm considering what to say and started a PR 625
Need to add clarification whether the users Setup was the anaconda, miniforge, or environment version, because that can affect what instructions to add here.
